CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# OpenTripPlanner Contributing Guide
+
+Thank you for your interest in contributing to OpenTripPlanner. This document will give you some pointers on how to contact and coordinate with the OTP development team, as well as a description of the contribution process and expectations.
+
+## Primary Channels of Communication
+
+If you have any questions about problems you are encountering with code, deployment, documentation, or development coordination please don't hesitate to post to the OpenTripPlanner discussion groups. These are Google Groups which can be accessed as web forums or as traditional email mailing lists:
+
+- https://groups.google.com/g/opentripplanner-dev (opentripplanner-dev@googlegroups.com)
+- https://groups.google.com/g/opentripplanner-users (opentripplanner-users@googlegroups.com)
+
+Any message posted there will be seen by most of the contributors, some of whom work on OTP full time. It will also create a record of the discussion that will be useful to the larger community and often leads to issues being discussed at the twice-weekly development meetings.
+
+OTP development meetings occur every Tuesday at 10 AM Central European time and Thursday at 15:00 Central European Time on https://meet.google.com/qwh-xzbz-atr. These meetings are open to anyone who wants to join, even if you simply want to observe the process or ask a few questions. The meeting times have been deliberately chosen to allow participation during regular business hours across as many time zones as possible, from the US (east coast) through Europe and Asia. 
```

Comment:
We had a discussion about this yesterday. We will change the meeting link to be always static (currently we have had to use different link if the host is not able to join). It can still be debated if we should define the meeting times in text here and/or use an online calendar that can be updated when, in the rare cases, we have to reschedule a meeting.

Reply:
I think this describe the current situation; hence can be merged. When we improve we will of cause update the document. I agree with @optionsome that we should aim for an online calendar (and link to it from here).

Reply:
The calendar URL is:

-  https://calendar.google.com/calendar/embed?src=ormbltvsqb6adl80ejgudt0glc%40group.calendar.google.com&ctz=Europe%2FOslo

Reply:
The ical URL to import the calendar is https://calendar.google.com/calendar/ical/ormbltvsqb6adl80ejgudt0glc@group.calendar.google.com/public/basic.ics